src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -263,6 +269,65 @@ export class HaConfigDevicePage extends LitElement {
     }
   }
 
+  private _renderDeleteButtons() {
+    console.log("Hello!")
+    const device = this._device(this.deviceId, this.devices);
+
+    if (!device) {
+      return;
+    }
+
+  let buttons = this._integrations(device, this.entries).map((entry) => {
+      console.log("Hello!")
+      console.log(entry.supports_remove_device)
+      if (entry.state !== "loaded" || !entry.supports_remove_device) {
+        return false;
+      }
+      return {
+        entry_id: entry.entry_id,
+        domain: entry.domain,
+      };
+    });
+
+    buttons = buttons.filter(Boolean);
+
+    if (buttons.length > 0) {
+      this._deleteButtons = (
+        buttons as { entry_id: string; domain: string }[]
+      ).map(
+        (button) => html`
+          <mwc-button class="warning" @click=${this._confirmDeleteEntry}>
```

Comment:
We should pass in the specific config entry id from `button.entry_id`, how to do that?